src/main.cpp

```diff
@@ -636,7 +662,7 @@ void tick_all_systems(const SysTick_s &current_system_tick)
     drivetrain.tick(current_system_tick);
     // // tick torque controller mux
 
-    DrivetrainCommand_s controller_output = case_system.evaluate(
+    auto _ = case_system.evaluate(
```

Comment:
Wait is a variable called underscore? Is there some C++ convention for this?

Reply:
![image](https://github.com/user-attachments/assets/c2ece7b4-ea04-4843-beae-c0872174afac)

its just a programming convention, however i will switch to using `__attribute__((unused))` instead as its a safer alternative